onedal/linear_model/linear_model.py

```diff
@@ -224,3 +234,92 @@ def fit(self, X, y, queue=None):
             self.intercept_ = self.intercept_[0]
 
         return self
+
+
+class Ridge(BaseLinearRegression):
+    """
+    Ridge Regression oneDAL implementation.
+
+    Parameters
+    ----------
+    alpha : float, default=1.0
+        Regularization strength; must be a positive float. Regularization
+        improves the conditioning of the problem and reduces the variance of
+        the estimates. Larger values specify stronger regularization.
+
+    fit_intercept : bool, default=True
+        Whether to calculate the intercept for this model. If set
+        to False, no intercept will be used in calculations
+        (i.e. data is expected to be centered).
+
+    copy_X : bool, default=True
+        If True, X will be copied; else, it may be overwritten.
```

Comment:
copy_X is used weirdly in LinearRegression.fit, but goes unused in your Ridge. You may have to change the documentation for this

Reply:
thanks for catch, now using copy_X in Ridge